annotation/tests/test_annotations.py

```diff
@@ -150,6 +160,93 @@ def annotation_doc_for_save():
     )
 
 
+@pytest.fixture
+def annotated_doc_schema(annotator: User):
+    yield AnnotatedDocSchema(
+        revision="20fe52cce6a632c6eb09fdc5b3e1594f926eea69",
+        user=annotator.user_id,
+        pipeline=None,
+        date=datetime(2024, 1, 1, 10, 10, 0),
+        file_id=1,
+        job_id=1,
+        pages={
+            "1": "19fe52cce6a632c6eb09fdc5b3e1594f926eea69",
+            "2": "adda414648714f01c1c9657646b72ebb4433c8b5",
+        },
+        validated={1, 2, 10},
+        failed_validation_pages={3, 4},
+        tenant="badger-doc",
+        task_id=2,
+        similar_revisions=None,
+        categories={"1", "2"},
+        links_json=[
+            {"to": 2, "category": "my_category", "type": "directional"}
+        ],
+    )
+
+
+@pytest.fixture
+def page_revision_list_all(annotated_doc_schema: AnnotatedDocSchema):
```

Comment:
this fixture and page_revision_list_latest are almost identical - it becomes overwhelming quite fast. Please leave only one fixture and make small testing data changes as you go. Fixtures are made to simplify testing by reusing data/functionality. In this case it just made things more complex

Reply:
I used different fixtures, since in the tests they represent different structures from the main.py: `PageRevision `and `LatestPageRevision`